src/kirby/components/list/list.component.ts

```diff
@@ -40,14 +39,8 @@ export class ListComponent implements OnChanges {
    * Provide items for the list to render. Items must be provided in the order you expect them to be rendered.
    */
 
-  private _items: ListItem[];
-  public get items(): ListItem[] {
-    return this._items;
-  }
   @Input()
-  public set items(items: ListItem[]) {
-    this._items = items ? this.addInternalId(items) : null;
-  }
+  public items: ListItem[];
```

Comment:
Revert to definition before your changes:
```suggestion
  public items: any[];
```